enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -165,8 +165,7 @@ type PebbleCache struct {
 	atimeBufferSize      int
 	minEvictionAge       time.Duration
 
-	lastDBVersionUpdate time.Time
-	lastDBVersion       filestore.PebbleKeyVersion
+	lastDBVersion filestore.PebbleKeyVersion
```

Comment:
The lastDBVersionUpdate was a vestige of something removed in a previous CL and unused, so I removed it.